test/integration/migrate/index.js

```diff
@@ -192,6 +192,10 @@ module.exports = function(knex) {
                 'fileName',
                 '20150109002832_invalid_migration.js'
               );
+          })
+          .then(function(data) {
+            // Clean up lock for other tests
+            return knex('knex_migrations_lock').update({ is_locked: 0 });
```

Comment:
any ideas why this only remains locked for Oracle? I wonder if something with transactions is wrong

Reply:
@kibertoad sometimes this is failing sometimes not, not very consistent.

Reply:
@kibertoad if you look this line I believe we need a cleanup here
https://github.com/tgriesser/knex/blob/master/test/integration/migrate/index.js#L177-L195

Reply:
I was also suspecting that oracle's transaction code has something done wrong. I tried to look into it some months ago, but wasn't able to reproduce nor fix it. 

It would be important to still add comment about how to make testbench to fail again to be able to really fix the underlying problem in oracle. And to add bug report issue to github pointing to disabled test.

Reply:
Done. Could you please check if this is what you've had in mind or if we are still missing something?